# Description: Triggers the detection of the given box model.

# A valid pose_frame (either camera or external) is required in the request.
# It can be provided either with the pose_frame field, as pose_frame of the selected region of interest
# or as pose_frame of the selected load carriers.
# If more than one pose_frame is provided, they must to be consistent.

# Optional. Pose frame for output poses.
string pose_frame

# Optional
string region_of_interest_id

# Optional
string load_carrier_id

# Optional
Compartment load_carrier_compartment

# Required
ItemModel[] item_models

# Required only if reference frame = external and sensor is robot-mounted
geometry_msgs/Pose robot_pose

---

time timestamp

Item[] items

LoadCarrier[] load_carriers

rc_common_msgs/ReturnCode return_code
